src/altinn-app-frontend/src/features/form/containers/RepeatingGroupsTable.test.tsx

```diff
@@ -14,21 +17,25 @@ import type { ILayoutState } from 'src/features/form/layout/formLayoutSlice';
 import type { IAttachments } from 'src/shared/resources/attachments';
 import type { IOption, ITextResource } from 'src/types';
 
-describe('RepeatingGroupTable', () => {
-  let mockContainer: ILayoutGroup;
-  let mockLanguage: any;
-  let mockTextResources: ITextResource[];
-  let mockAttachments: IAttachments;
-  let mockComponents: ILayoutComponent[];
-  let mockLayout: ILayoutState;
-  let mockCurrentView: string;
-  let mockData: any;
-  let mockOptions: IOption[];
-  let repeatingGroupIndex: number;
-  let mockRepeatingGroupDeepCopyComponents: Array<
-    Array<ILayoutComponent | ILayoutGroup>
-  >;
+import type { ILanguage } from 'altinn-shared/types';
+
+const user = userEvent.setup();
 
+let mockContainer: ILayoutGroup;
+let mockLanguage: ILanguage;
+let mockTextResources: ITextResource[];
```

Comment:
I know you probably just continued on existing code here, but is there really a need to declare these variables before each run? I'm assuming they never actually change.

Could we not just declare them as const and remove the `beforeEach` all together? :)